AppCenter/AppCenter/Internals/Util/MSPerformSelectorUtil.h

```diff
@@ -3,9 +3,70 @@
 
 #import <Foundation/Foundation.h>
 
+#define ARRAY_FROM_ARGS(...)                                                                                                               \
+  ({                                                                                                                                       \
+    NSMutableArray *initedArray = [NSMutableArray arrayWithObjects:[NSNull null], ##__VA_ARGS__, nil];                                     \
+    [initedArray removeObjectAtIndex:0];                                                                                                   \
+    initedArray;                                                                                                                           \
+  })
```

Comment:
I guess it won't work. I don't see `\` at the end of lines. Did you tested it?

Reply:
clang-format-changed-files moved this to the rightmost edge of the line, it's work